### PR TITLE
fixes for cypress tests

### PR DESCRIPTION
**Pull request by RishabhKodes**
Current status for all the Cypress tests:
- active-health-check 
 image
- add-capacity-spec 
image
- expand-pvc
image
- list-page  
image
- multiple-pool  
image
- namespace-store 
image

### CODE REVIEW

File: cypress/tests/add-capacity.spec.ts [hunk 1]
```diff
@@ -90,19 +90,19 @@ describe('OCS Operator Expansion of Storage Class Test', () => {
       // Todo(bipuladh): Add a proper data-selector once the list page is migrated
       // eslint-disable-next-line cypress/require-data-selectors
       cy.get('a').contains(STORAGE_SYSTEM_NAME).should('exist');
-      cy.byLegacyTestID('kebab-button').click();
-      cy.byTestActionID('Add Capacity').click();
+      // eslint-disable-next-line cypress/no-unnecessary-waiting
+      cy.wait(2000);
```
Comment (bipuladh) [inline cypress/tests/add-capacity.spec.ts:94]: Do not use arbitrary wait times.

File: cypress/tests/add-capacity.spec.ts [hunk 2]
```diff
@@ -90,19 +90,19 @@ describe('OCS Operator Expansion of Storage Class Test', () => {
...
+      cy.byTestID('kebab-button').click();
+      // eslint-disable-next-line cypress/require-data-selectors
+      cy.get('a').contains('Add Capacity').click();
```
Comment (bipuladh) [inline cypress/tests/add-capacity.spec.ts:97]:
```suggestion
      // eslint-disable-next-line cypress/require-data-selectors
      cy.contains('Add Capacity').click();
```

File: cypress/tests/list-page.spec.ts [hunk 3]
```diff
@@ -19,11 +19,13 @@ describe('Tests storage system list page', () => {
...
   it('Test default(OCS) StorageSystem is listed', () => {
+    ODFCommon.visitStorageDashboard();
     ODFCommon.visitStorageSystemList();
     listPage.searchInList(STORAGE_SYSTEM_NAME);
     // Test if the Kebab Menu contains all Items
-    cy.byLegacyTestID('kebab-button').click();
-    cy.byTestActionID('Add Capacity').click();
+    cy.byTestID('kebab-button').click();
+    // eslint-disable-next-line cypress/require-data-selectors
+    cy.get('a').contains('Add Capacity').click();
```
Comment (bipuladh) [inline cypress/tests/list-page.spec.ts:28]:
```suggestion
    // eslint-disable-next-line cypress/require-data-selectors
    cy.contains('Add Capacity').click();
```

File: packages/odf/components/system-list/odf-system-list.tsx [hunk 4]
```diff
@@ -150,7 +150,7 @@ const tableColumnInfo = [
     id: 'throughput',
   },
   { className: classNames('pf-m-hidden', 'pf-m-visible-on-xl'), id: 'latency' },
-  { className: 'dropdown-kebab-pf pf-c-table__action', id: '' },
+  { className: 'dropdown-kebab-pf pf-c-table__action', id: 'kebab-button' },
```
Comment (bipuladh) [inline packages/odf/components/system-list/odf-system-list.tsx:153]:
```suggestion
  { className: 'dropdown-kebab-pf pf-c-table__action'},
```
Use `data-test` directly.

File: cypress/views/modals.ts [hunk 5]
```diff
@@ -13,7 +13,8 @@ export const modal = {
   cancel: (force: boolean = false) =>
     cy.byLegacyTestID('modal-cancel-action').click({ force }),
   // eslint-disable-next-line cypress/require-data-selectors,cypress/no-force
-  submit: (force: boolean = false) => cy.get(submitButton).click({ force }),
+  submit: (force: boolean = false) =>
+    cy.get('button').contains('Add').click({ force }),
```
Comment (bipuladh) [inline cypress/views/modals.ts:17]:
```suggestion
    cy.contains('Add').click({ force }),
```

**Review by bipuladh — COMMENTED** [review]
Use data-test if you can't get certain elements. 
For ex: 
```
cy.get('a')
```
should be: 
```
 cy.byTestID('add-capacity-button').click();
```

File: cypress/views/modals.ts (hunk 5 above)
Comment (RishabhKodes) [reply cypress/views/modals.ts:17]: This function has to be in a Single line according to prettier.

Comment (bipuladh) [reply cypress/views/modals.ts:17]: Sorry I am confused. The comment is about removing the unnecessary `get`

Comment (RishabhKodes) [reply cypress/views/modals.ts]: Not able to locate the button without the `get(button)` command.

File: cypress/tests/active-health-checks.ts [hunk 6]
```diff
@@ -1,7 +1,7 @@
-import { scaleDeployments } from '../helpers/active-health-checks';
 import { MINUTE, STORAGE_SYSTEM_NAME } from '../consts';
-import { ODFCommon } from '../views/odf-common';
+import { scaleDeployments } from '../helpers/active-health-checks';
 import { listPage } from '../views/list-page';
+import { ODFCommon } from '../views/odf-common';
```
[the remark this answers is not in the document]
Comment (RishabhKodes) [reply cypress/tests/active-health-checks.ts:4]: Just fixing the order of Imports.

File: cypress/tests/list-page.spec.ts [hunk 7]
```diff
@@ -19,11 +19,13 @@ describe('Tests storage system list page', () => {
...
 
   it('Test default(OCS) StorageSystem is listed', () => {
+    ODFCommon.visitStorageDashboard();
     ODFCommon.visitStorageSystemList();
     listPage.searchInList(STORAGE_SYSTEM_NAME);
     // Test if the Kebab Menu contains all Items
-    cy.byLegacyTestID('kebab-button').click();
-    cy.byTestActionID('Add Capacity').click();
+    cy.byTestID('kebab-button').click();
+    // eslint-disable-next-line cypress/require-data-selectors
```
[the remark this answers is not in the document]
Comment (RishabhKodes) [reply cypress/tests/list-page.spec.ts:27]: Ack, updated the code. Will push commit after e2e tests are finished running.

**Comment by vbnrh** [comment]
Make the change in `views/common.ts` from 'OpenShift Data Foundation` to `Data Foundation`. This is one of the failures i noticed.

**Comment by RishabhKodes** [comment]
> Make the change in `views/common.ts` from `OpenShift Data Foundation` to `Data Foundation`. This is one of the failures i noticed.

@vbnrh It was already done here, https://github.com/red-hat-storage/odf-console/pull/201/commits/f4aa427fe036ba52d5324d70bdc404122b79b3ea

**Pull request merged**